ocaml/testsuite/tests/jane-modular-syntax/attribute_locations.ml

```diff
@@ -0,0 +1,70 @@
+(* TEST
+   include ocamlcommon *)
+
+let () = Language_extension.enable Comprehensions ();;
+
+ module Location_map = struct
+   include Map.Make (struct
+     type t = Location.t
+     let compare = compare
+  end)
+
+   let add_multi key data t =
+     update key (function
+       | None -> Some [data]
+       | Some x -> Some (data :: x))
+       t
+end
+
+let gather_attributes_by_location program_text =
+  let program_text_buf = Lexing.from_string program_text in
+  Lexing.set_filename program_text_buf "<no filename in test>";
```

Comment:
I got fooled by the test output into thinking "`<no filename in test>`" was the bad thing.  Maybe just `test.ml` or `dummy_file.ml` would be clearer?

Reply:
I can definitely see how this output might be confusing in the context of reviewing this change, but I think it's fine output to merge into `main`. I think it will be clear enough that "<no filename in test>" is the good thing by virtue of the fact that it's the only thing that appears in this output.